website/developer-docs/docs/style-guide.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: Style Guide
+---
+
+Here are a few guidelines to ensure that the documentation is easy to read and uses similar phrasing. Remember to use our [docs templates](./templates/index.md) when possible; they are already set up to follow may of the style guidelines below.
+
+If you find any documentation that doesn't match these guidelines, feel free to either open an [Issue](https://github.com/goauthentik/authentik/issues) or a [PR](https://github.com/goauthentik/authentik/pulls) so they can be fixed.
+
+To test how the documentation renders use the Netlify Deploy Preview, especially when using Docusaurus-specific features. Or [build it locally and test](./writing-documentation.md#setup-your-local-build), using the `make website-watch` command.
+
+:::info
+Make sure to add any new pages to the `sidebar` file. Otherwise, the new page will not appear in the table of contents to the left.
+:::
+
+## General guidelines
+
+-   Try to order the documentation sections in an order that makes it easiest for the user to configure. That is, order the sections in the same way the actual workflow to accomplish the task is ordered. Use headings (sub-titles) to break up long documents, and make it easier to find a specific section.
+
+-   You can use standard [Docusaurus-specific features](https://docusaurus.io/docs/next/markdown-features), which include MDX elements such as tabs and admonitions.
+
+## Terminology
+
+### authentik names
+
+-   The product name authentik should always start with a lower-case "a" and end with a "k". Even if it is the first word in a sentence. :-)
+
+-   Our company name is Authentik Security, Inc. but in non-legal docs you can shorten it to Authentik Security.
+
+### Industry terms, technology, and other tools
+
+-   When referring to external tools, or an industry term or technology, always follow the exact capitalization that the product or company itself uses, or what the industry uses in consensus.
+
+-   Use abbreviations where it makes sense (for commonly used terms like SAML and OAuth) for common terms. If an abbreviation is less-known, spell it out in parentheses after the first use.
```

Comment:
`(for commonly used terms like SAML and OAuth) for common terms` is a bit redundant imo